qemu/lib/iozone.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import os
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import postprocess_iozone
+
+
+class _IozoneTest(object):
+    """
+    Prepare iozone env, run iozone on guest, clean up iozone env.
+    """
+    def __init__(self, vm, params, test):
+        """
+        run setup of iozone test
+        """
+        self.params = params
+        self.session = vm.wait_for_login()
+        self.test = test
+        self.results_path = ""
+        self.timeout = float(params.get("iozone_timeout", 240))
+
+    def before_run_iozone(self):
+        """
+        Prepare for iozone test
+        """
+        raise NotImplementedError()
+
+    def run_iozone_in_guest(self, mountpoint):
+        """
+        Run iozone test in guest
+
+        :param mountpoint: mountpoint/disk letter assigned to disk under test
+        :return: iozone run status and result.
+        """
+        iozone_test_cmd_default = "./iozone -azR -r 64k -n 125M -g 512M -i 0 -i 1 %s"
+        iozone_test_cmd = self.params.get("iozone_test_cmd", iozone_test_cmd_default)
```

Comment:
If you want user customize the it, the more accurate name of the param should be `iozone_options`, since user do not know where `iozone` located in.